eng/Packages.Data.props

```diff
@@ -282,7 +282,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <TestProxyVersion>1.0.0-dev.20221111.1</TestProxyVersion>
+    <TestProxyVersion>1.0.0-dev.20230131.4</TestProxyVersion>
```

Comment:
@scbedd I thought we controlled this version in eng/common. Is that not true for .NET?

Reply:
@weshaggard we do. Trying to figure wtf is going on with updating the recordings.

90% of the time _one_ or more of the recordings will randomly fail with a 400. We know it's proxy related, but at this point I have NO IDEA where it's coming from.